src/visualization.jl

```diff
@@ -166,9 +166,9 @@ function Base.open(controls::SimulationControls, window::Window)
     body!(window, render_default(controls))
 end
 
-function SimulationStatus(controls::SimulationControls{<:Widget{:button}})
-    terminate = map(!iszero, observe(controls.terminate))
-    pause = map(isodd, observe(controls.pause))
```

Comment:
This doesn't look right. The extra `Observable{Bool}(false)` shouldn't be necessary, and we need to call `observe()` on both controls. I'm not sure why this isn't working on your system, but it's been fine for me. I'm probably just on a strange combination of package versions, so I'll try to work it out.

Reply:
It's https://github.com/JuliaGizmos/Observables.jl/pull/15.